### DISCUSSION TITLE
Managed Identity Support for Office 365 connectors

### DISCUSSION THREAD

**Discussion by divyaswarnkar** [Azure/logicapps#633] [Feature Ideas/Requests]
If you have a use case for Managed Identity support with Office 365 connectors please share more details with us. We have heard requests for Outlook, Teams to support MI. Currently Office 365 doesn't support Managed Identity. We are collecting this information to facilitate our discussions with them and provide your needs as a data to support this request.

**Comment by DanielMeixner** [comment]
My customer wants to connect from Logic Apps to Teams using the Teams connector to post a message to a channel. At this point you have to use a personal identity to connect. This is very unpleasant for several reasons: It's hard to put this into IaC (Terrafrom/Bicep) and in general a bad idea to have a personal identity used for a service. It would be great if you'd offer managed identity support here.

**Comment by joerodgers** [comment]
+1 to @DanielMeixner's comments. My customer has a lot of Logic App based automation tasks which connect to a shared mailbox in EXO to send automated emails and they are running into the same challenges stated above.

**Comment by itsmecurtis** [comment]
We're using the "Word Online (Business)" connector to populate a Word template with values, convert the result to .pdf, and store it in Azure Storage and the requirement to perform interactive authentication against an end-user account was surprising.  Please support Managed Identity for this scenario. Thanks!
